Translators/PlannedEvents/README.md

```diff
@@ -0,0 +1,53 @@
+# Planned Events Translator
+
+## Table of Contents
+
+- [About](#about)
+- [Getting Started](#getting_started)
+- [Usage](#usage)
+- [Contributing](../CONTRIBUTING.md)
+
+## About <a name = "about"></a>
+
+Python Planned Events to TIM message translator that is designed to pull messages from the CDOT Planned Events feed and translate to TIM messages.

```

Comment:
suggestion: clarify that we're only interested in the "closed for the season" messages as the rest are already handled by the WZDx feed.

Reply:
suggestion: include sample input/output files

Reply:
I've updated this to include that we're only interested in 'closed for the season' messages and added links to documentation for the feed.